pkg/checks/external/main.go

```diff
@@ -1199,7 +1199,7 @@ func (ext *Checker) podExists() (bool, error) {
 
 	// if the pod is "not found", then it does not exist
 	p, err := podClient.Get(ext.podName(), metav1.GetOptions{})
-	if err != nil && k8sErrors.IsNotFound(err) {
```

Comment:
this is statement wasn't correctly assessing "not found" pods -- so i reverted back to how we were previously determining if pods did / didn't exist